seed/models/building_file.py

```diff
@@ -57,7 +58,7 @@ class BuildingFile(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
     property_state = models.ForeignKey('PropertyState', on_delete=models.CASCADE, related_name='building_files', null=True)
-    file = models.FileField(upload_to="buildingsync_files", max_length=500, blank=True, null=True)
+    file = models.FileField(upload_to="media/buildingsync_files" if settings.USE_S3 is True else "buildingsync_files", max_length=500, blank=True, null=True)
```

Comment:
Can you confirm that with this setting, S3 uploads to `media/buildingsync_files` and not `media/media/buildingsync_files`?

Reply:
Yes. I can confirm it uploads to media/buildingsync_files in the s3 bucket.